opossum-tool/OpossumUI
# Add feature to directly open a file via ENV variable

### Summary of changes

In addition to opening a file from a cli argument this commit adds the function to open a file by setting an environment variable. If the ENV variable OPOSSUM_FILE is set, the app will try to open this file directly when starting. To simplify the usage of this feature and make it independent of the operating system the yarn start command now accepts an optional argument and sets the environment variable when starting the app.
### Context and reason for change

With these changes the app can be started with `yarn start` (using vite) and directly open an opossum input file.
### How can the changes be tested

See the additional tests. 
Alternatively, run `yarn start <input_file>` where `<input_file>` is the absolute path to the file which should be opened. The app is now started and directly opens the provided file.
